src/Stripe.Tests/transfers/test_data/stripe_transfer_create_options.cs

```diff
@@ -18,7 +18,8 @@ public static StripeTransferCreateOptions Valid(string recipient = "self")
                 {
                     { "A", "Value-A" },
                     { "B", "Value-B" }
-                }
+                },
+                ApplicationFee = 10
```

Comment:
Alternatively, you could just copy/paste a test as a base, then get a Valid object, and set the application fee in that test, then assert in that test. Make sense?